src/types/NFTInterface.ts

```diff
@@ -254,12 +254,20 @@ export type NFTObject = {
     component: string;
   };
   media?: {
-    thumbnail?: MediaObject;
+    source: MEDIA_SOURCES;
     image?: MediaObject;
     large?: MediaObject;
     poster?: MediaObject;
+    thumbnail?: MediaObject;
+    // backwards compatibility
     content?: MediaObject;
```

Comment:
We probably? safely can remove this content field?

Reply:
so idea is to retain backwards compatibility
i was thinking about going back to it when covering the whole thing with tests, wdyt?

Reply:
sounds good